### PR TITLE
Customization wrappers

### PR DESCRIPTION
**Pull request by yuki-stripe**
## Summary
* Add wrappers around STDSUICustomization classes
* Fix STPThreeDSCustomizationSettings init (https://jira.corp.stripe.com/browse/MOBILE3DS2-404)
* Bumps stripe-3ds2-ios submodule to latest master

## Tests
Added tests.  Would like to also test this out in ios-3ds2-dogfooding, but need to figure out how to update that project with these changes.

In a future PR I will add protocols for all theses customization classes in Stripe3DS2 and conform the classes here.

### CODE REVIEW

File: Stripe/Payments/STPThreeDSTextFieldCustomization.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ A customization object to use to configure the UI of a text field.
+ */
+@interface STPThreeDSTextFieldCustomization : NSObject
+
+/// The default settings.
++ (instancetype)defaultSettings;
+
+/// The border width of the text field. Defaults to 2.
```
Comment (csabol-stripe) [inline Stripe/Payments/STPThreeDSTextFieldCustomization.h]: is this points or pixels? Should it be a CGFloat instead?

File: Stripe/PublicHeaders/STPThreeDSCustomizationSettings.h [hunk 2]
```diff
@@ -27,12 +27,16 @@ NS_ASSUME_NONNULL_BEGIN
  `uiCustomization` can be used to provide custom UI settings for the authentication
  challenge screens presented during a Three Domain Secure authentication. For more information see
  our guide on supporting 3DS2 in your iOS application.
+ 
+ Defaults to `[STDSUICustomization defaultSettings]`.
```
Comment (csabol-stripe) [inline Stripe/PublicHeaders/STPThreeDSCustomizationSettings.h]:
```suggestion
 Defaults to `[STPThreeDSUICustomization defaultSettings]`.
```

File: Stripe/PublicHeaders/STPThreeDSCustomizationSettings.h [hunk 3]
```diff
@@ -27,12 +27,16 @@ NS_ASSUME_NONNULL_BEGIN
...
  */
-@property (nonatomic, nullable) STDSUICustomization *uiCustomization;
+@property (nonatomic) STPThreeDSUICustomization *uiCustomization;
 
 /**
  `authenticationTimeout` is the total time allowed for a user to complete a 3DS2 authentication
  interaction. This value *must* be at least 5 minutes.
+ 
+ Defaults to 10 minutes.
```
Comment (csabol-stripe) [inline Stripe/PublicHeaders/STPThreeDSCustomizationSettings.h]: I'm not sure why I made this 10 minutes in the first place. Should probably go with 5

File: Stripe/Payments/STPThreeDSTextFieldCustomization.h (hunk 1 above)
Comment (yuki-stripe) [reply Stripe/Payments/STPThreeDSTextFieldCustomization.h]: Ah I missed this one, should be CGFloat for sure.  Will follow up.

**Pull request merged**